Source/relay/Guide/Items of the Month/Pizza Cube.ash

```diff
@@ -10,12 +10,17 @@ void IOTMPizzaCube(ChecklistEntry [int] resource_entries)
 
         // Entries
         string [int] description;
+        
+        boolean need_cheese = $item[goat cheese].available_amount() <3; // Obviously incomplete
```

Comment:
Can we update this to also not show up if we've gotten past that point in the quest?